day1.md

```diff
@@ -0,0 +1,119 @@
+
+We had been starving and this led us to believe there were a lot of great restaurants outside of the station waiting for us. The station was huge; it took us quite a while to figure out how to get out. Long story short, after resisting all the temptations from foods inside the station and stumbling our ways through, we finally got out and there was nothing but office buildings and a mall with restaurants opening in about two hours.
+<img src="day1/14379890_1115746348475058_5032397402719858860_o.jpg" width="500"><br/>
+<img src="day1/14352256_1115746385141721_8737827987627050363_o.jpg" width="500"><br/>
+
+Matt and I were, in Katherine's words, so grumpy that it scared her. We got back to the station, failed to navigate back to the part packed with food shops, ended up getting Ekibenes from a small shop, got back to the mall and ate there because the station was so crowded that there was no place for us to sit and eat.
+
+<img src="day1/IMG_20160904_214125.jpg" width="500"><br/>
+Here comes my first <a name="first_meal">meal</a> in Japan! Looking at the picture still makes me hungry now 😂
+
+After resting a bit in the mall, we went to the post office. Man, look at all these postcards! 😱
+<img src="day1/IMG_20160904_224447.jpg" width="500"><br/>
+<img src="day1/IMG_20160904_224508.jpg" width="500"><br/>
+<img src="day1/IMG_20160904_224503.jpg" width="500"><br/>
+They even have series of postcards for trains 😱
+
+Then we headed to <a name="ikebukuro_station">Ikebukuro Station</a> because the place we were staying for the next three nights, booked through [Airbnb]((https://www.airbnb.com/rooms/4585434)), was about 15 mins from the station.
+
+There was a small cheese tart shop with a long line waiting outside at Ikebukuro Station.
+<img src="day1/IMG_20160904_233744.jpg" width="500"><br/>
+<img src="day1/IMG_20160904_233807.jpg" width="500"><br/>
+<img src="day1/14372351_1115746725141687_2298785123460181227_o.jpg" width="500"><br/>
+Hot cheese inside 🔥
+
+On our way to the Airbnb place.<br/>
+<img src="day1/14352021_1115746868475006_8883205870417679098_o.jpg" width="500"><br/>
+<img src="day1/14409899_1115746961808330_4583957709311564211_o.jpg" width="500"><br/>
+
+After checking-in and dropping our bags at our Airbnb place, we were on our way to <a name="odaiba_onsen">Odaiba Onsen</a>, the main event of our first day in Japan!
+<br/><img src="day1/14362487_1115747058474987_3909970043623033642_o.jpg" width="500"><br/>
+
+There was a ~25 mins walk<sup>[4](#walk)</sup> from Teleport Station to Odaiba Onsen. It felt like a pretty long walk for us, mainly because we were tired out by the hot weather 🔥. We even had a small break near the end of the walk and got some foods at a Family Mart, one of the three most common convenient stores in Japan.
+
+We were pretty excited when we finally arrived and gonna spend the rest of the day in door!
```

Comment:
Use "and were gonna" instead of "and gonna." (We'd say, "when we...were gonna spend.")

Also "indoors" instead of "in door" :)